packages/mediacenter/kodi/package.mk

```diff
@@ -203,7 +205,7 @@ KODI_LIBDVD="$KODI_DVDCSS \
              -DLIBDVDREAD_URL=$SOURCES/libdvdread/libdvdread-$(get_pkg_version libdvdread).tar.gz"
 
 # Build Kodi using parallel LTO
-[ "$LTO_SUPPORT" = "yes" -a "$DEBUG" != "yes" ] && PKG_KODI_USE_LTO="-DUSE_LTO=$CONCURRENCY_MAKE_LEVEL"
+[ "$LTO_ENABLED" = "yes" -a "$DEBUG" != "yes" ] && PKG_KODI_USE_LTO="-DUSE_LTO=$CONCURRENCY_MAKE_LEVEL"
```

Comment:
This won't work as expected (ie. reliably), since LTO_ENABLED won't be set until _after_ the package is sourced. You might find it's not set the first time the package is sourced in `config/path` but would be set by the time the package is sourced a second time in `scripts/build`. One solution would be to put this all inside a `configure_package`.

Also (and this is my error) the test for `$DEBUG != "yes"` needs fixing now that we can selectively build with debug - this should be changed to `"${DEBUG:-no}" = "no"`.